justfile

```diff
@@ -298,3 +271,97 @@ bootstrap *ARGS:
             --no-owner \
             --verbose \
             /code/{{ file }}
+
+# Clear sessions
+[group('database')]
```

Comment:
Maybe this could be either in server (as the command runs on the production server -- fab production) or in the django (as it runs a django management command internally) group?